pyproject.toml

```
@@ -138,18 +138,13 @@ dev = [
     "black==24.4.2",
     "isort==5.13.2",
     "mypy==1.10.0",
-    "psutil==5.9.2",
-    "stix2==3.0.1",
-    "requests==2.31.0",
     "mypy-protobuf==3.6.0",
     # type stubs for mypy
     "types-backports==0.1.3",
     "types-colorama==0.4.15.11",
     "types-PyYAML==6.0.8",
     "types-tabulate==0.9.0.20240106",
     "types-termcolor==1.1.4",
```

```suggestion
    "types-termcolor==1.1.4",
    "types-psutil==5.8.23",
    "types_requests==2.32.0.20240602",
```

re-add the types for dev (mypy)